p11-kit/rpc-server.c

```diff
@@ -2761,7 +2761,7 @@ p11_kit_remote_serve_tokens (const char **tokens,
 				goto out;
 			}
 			p11_virtual_init (lower, &p11_virtual_base, module, NULL);
-			filter = p11_filter_subclass (lower, NULL);
+			filter = p11_filter_subclass (lower, free);
```

Comment:
Isn't it sufficient to supply `free` as the `lower_destroyer` parameter of `p11_virtual_init` above?

Reply:
Just tried. It is not.